src/sentry/apidocs/examples/project_examples.py

```diff
@@ -110,6 +110,324 @@
+            "profile-file-io-main-thread-ingest",
+            "customer-domains",
+            "performance-file-io-main-thread-post-process-group",
+            "issue-alert-fallback-targeting",
+            "performance-render-blocking-asset-span-visible",
+            "ds-sliding-window-org",
+            "performance-consecutive-http-ingest",
+            "profile-image-decode-main-thread-post-process-group",
+            "performance-mep-bannerless-ui",
+            "performance-uncompressed-assets-visible",
+            "getting-started-doc-with-product-selection",
+            "performance-large-http-payload-visible",
+            "performance-view",
+            "promotion-mobperf-discount20",
+            "performance-new-widget-designs",
+            "profile-json-decode-main-thread-visible",
+        ],
+        "links": {
+            "organizationUrl": "https://sentry.sentry.io",
+            "regionUrl": "https://us.sentry.io",
+        },
+        "hasAuthProvider": True,
+    },
+    "plugins": [
+        {
+            "id": "asana",
+            "name": "Asana",
+            "slug": "asana",
+            "shortName": "Asana",
+            "type": "issue-tracking",
+            "canDisable": True,
+            "isTestable": False,
+            "hasConfiguration": True,
+            "metadata": {},
+            "contexts": [],
+            "status": "unknown",
+            "assets": [],
+            "doc": "",
+            "firstPartyAlternative": None,
+            "deprecationDate": None,
+            "altIsSentryApp": None,
+            "enabled": True,
+            "version": "23.7.0.dev0",
+            "author": {"name": "Sentry Team", "url": "https://github.com/getsentry/sentry"},
+            "isDeprecated": False,
+            "isHidden": False,
+            "description": "\nImprove your productivity by creating tasks in Asana directly\nfrom Sentry issues. This integration also allows you to link Sentry\nissues to existing tasks in Asana.\n",
+            "features": ["issue-basic"],
+            "featureDescriptions": [
+                {
+                    "description": "Create and link Sentry issue groups directly to an Asana ticket in any of your\n            projects, providing a quick way to jump from a Sentry bug to tracked ticket!",
+                    "featureGate": "issue-basic",
+                },
+                {
+                    "description": "Link Sentry issues to existing Asana tickets.",
+                    "featureGate": "issue-basic",
+                },
+            ],
+            "resourceLinks": [
+                {"title": "Report Issue", "url": "https://github.com/getsentry/sentry/issues"},
+                {
+                    "title": "View Source",
+                    "url": "https://github.com/getsentry/sentry/tree/master/src/sentry_plugins",
+                },
+            ],
+        }
+    ],
+    "platforms": ["native", "other", "python"],
```

Comment:
Again not related and just asking for my info. What is platform "other"? I don't see it on UI

Reply:
We have the key `platform` defined at the beginning of the example that's only what the user selected on the UI. I'm not sure how these other platforms get added to 'platforms' or what `other` is unfortunately